src/App.tsx

```diff
@@ -24,22 +23,24 @@ function App() {
   });
 
   useEffect(() => {
-    setIsLogedin(accessToken ? true : true);
-    console.log("isLogedin", isLogedin, accessToken);
-  }, []);
-
-  const [userProfile, setuserProfile] = useState<UserModal | null>();
+    setIsLogedin(accessToken ? true : false);
```

Comment:
Token will be the one received from api.

Reply:
Updated